### PR TITLE
[js/vertexai]: docs: how to add grounding to gemini

### PR DESCRIPTION
**Pull request by pr-Mais**
Updating the vertex ai plugin docs.

### CODE REVIEW

File: docs/plugins/vertex-ai.md [hunk 1]
```diff
@@ -96,6 +96,30 @@ const llmResponse = await generate({
...
+[Google Search](https://cloud.google.com/vertex-ai/generative-ai/docs/multimodal/ground-gemini#web-ground-gemini)
+and/or [your own data](https://cloud.google.com/vertex-ai/generative-ai/docs/multimodal/ground-gemini#private-ground-gemini):
+
+```js
+await generate({
+  model: gemini15Flash,
+  prompt: ...,
+  config: {
+    googleSearchRetrieval: {
+      disableAttribution: true,
```
Comment (pavelgj) [inline docs/plugins/vertex-ai.md:109]: Does this need to be set to true?

File: docs/plugins/vertex-ai.md [hunk 2]
```diff
@@ -96,6 +96,30 @@ const llmResponse = await generate({
 });
 ```
 
+This plugin also supports grounding Gemini text responses using
+[Google Search](https://cloud.google.com/vertex-ai/generative-ai/docs/multimodal/ground-gemini#web-ground-gemini)
```
Comment (pavelgj) [inline docs/plugins/vertex-ai.md:100]: This is not a cheap feature. Wonder if it's worth linking to pricing....

Comment (pr-Mais) [reply docs/plugins/vertex-ai.md:100]: Yeah I think it's important to add a highlighted note with the link, do you know if the following syntax can show as a note in the [Firebase page docs](https://firebase.google.com/docs/genkit/plugins/vertex-ai)? I think the docs there is fetched from the markdown files here?
> [!IMPORTANT]
> Please beware of the costs associated with groudnding responses with Google Search, check the [pricing](https://cloud.google.com/vertex-ai/generative-ai/pricing#gemini-models) page for more information.

Comment (pavelgj) [reply docs/plugins/vertex-ai.md:100]: Yes, something like this LGTM. Added @kevinthecheung to comment on the wording.
/cc @chrisraygill

Comment (kevinthecheung) [reply docs/plugins/vertex-ai.md:100]: I suggest:

```suggestion
Important: Vertex AI charges a fee for grounding requests in addition to the cost of making LLM requests.  See the [Vertex AI pricing](link) page and be sure you understand grounding request pricing before you use this feature.
```

DevSite will render it as a note

**Review by kevinthecheung — APPROVED** [review]

File: docs/plugins/vertex-ai.md [hunk 3]
```diff
@@ -96,6 +96,30 @@ const llmResponse = await generate({
 });
 ```
 
+This plugin also supports grounding Gemini text responses using
+Important: Vertex AI charges a fee for grounding requests in addition to the cost of making LLM requests.  See the [Vertex AI pricing](link) page and be sure you understand grounding request pricing before you use this feature.
+and/or [your own data](https://cloud.google.com/vertex-ai/generative-ai/docs/multimodal/ground-gemini#private-ground-gemini):
```
Comment (kevinthecheung) [inline docs/plugins/vertex-ai.md:101]:
```suggestion
This plugin also supports grounding Gemini text responses using
[Google Search](https://cloud.google.com/vertex-ai/generative-ai/docs/multimodal/ground-gemini#web-ground-gemini)
or [your own data](https://cloud.google.com/vertex-ai/generative-ai/docs/multimodal/ground-gemini#private-ground-gemini).

Important: Vertex AI charges a fee for grounding requests in addition to the cost of making LLM requests.  See the [Vertex AI pricing](link) page and be sure you understand grounding request pricing before you use this feature.

Example:
```

**Comment by MichaelDoyle** [comment]
Are we waiting on anything to merge this in?

**Comment by pr-Mais** [comment]
@MichaelDoyle no, should be ready to go.

**Review by pavelgj — APPROVED** [review]

**Pull request merged**